cypress/e2e/default/dynamic-routes.cy.ts

```diff
@@ -17,7 +19,11 @@ describe('Static Routing', () => {
   it('renders correct page via ODB on a static route', () => {
     cy.request({ url: '/getStaticProps/with-revalidate/', headers: { 'x-nf-debug-logging': '1' } }).then((res) => {
       expect(res.status).to.eq(200)
-      expect(res.headers).to.have.property('x-nf-render-mode', 'odb ttl=60')
+      if (CDNCacheControlEnabled) {
+        expect(res.headers).to.have.property('netlify-cdn-cache-control', 's-maxage=1, stale-while-revalidate')
+      } else {
+        expect(res.headers).to.have.property('x-nf-render-mode', 'odb ttl=60')
+      }
```

Comment:
I don't like test expectations that have logic in them as I feel it's harder to parse. What do you think about having a separate test suite that uses the same tests but enables the feature flag and this one disables it?

Reply:
Moving them to separate file make keeping things in sync harder and more manual work on trying to keep them in sync  (i.e. if we add new test case, we should add to both, but if you add to just one of them, nothing will signal that things are not in sync anymore).

While I'm not huge fan of doing what I did, I think it's realistically least bad option if we want have (and maintain) parity on what behaviors we test

Reply:
Makes sense. Thanks.